packages/tooling/fast-tooling/src/web-components/css-layout/css-layout.ts

```diff
@@ -139,6 +140,32 @@ export class CSSLayout extends FormAssociatedCSSLayout {
         /* eslint-disable-next-line @typescript-eslint/no-empty-function */
     }): void => {};
 
+    /**
+     * Handles the css-layout controls being enabled by pressing space
+     * @internal
+     */
+    public handleKeypressCSSChange = (
+        observablePropertyName: ObservableFlexboxCSSPropertyName,
+        e: KeyboardEvent
+    ): void => {
+        switch (e.keyCode) {
```

Comment:
This will preventDefault() on all keypresses, consider returning true for those you aren't handling.

Reply:
That's fine, that's a desirable outcome since this is all self contained, we don't expect any other events to be emitted than a change event fired from the component.